scripts/create-release-pr.sh

```
@@ -0,0 +1,184 @@
+    local head_ref="$2"
+
+    # Fetch if comparing against remote refs
+    if [[ "$base_ref" == origin/* ]]; then
+        local branch="${base_ref#origin/}"
+        git fetch origin "$branch" 2>/dev/null || true
+    fi
+
+    # Get list of changed Cargo.toml files in program-libs, sdk-libs, and program-tests/merkle-tree
+    for file in $(git diff "$base_ref"..."$head_ref" --name-only -- '**/Cargo.toml' | grep -E '(program-libs|sdk-libs|program-tests/merkle-tree)/'); do
+        # Extract old and new version from the diff
+        local versions=$(git diff "$base_ref"..."$head_ref" "$file" | grep -E '^\+version|^-version' | grep -v '+++\|---')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)